$NetBSD: patch-aa,v 1.1.1.1 2005/06/25 01:55:08 reed Exp $

these utilities were written with qt2 but they still ok with qt3

--- icewm4vim/Makefile.orig	2005-03-07 15:55:33.000000000 +0000
+++ icewm4vim/Makefile	2005-03-07 15:54:07.000000000 +0000
@@ -1,10 +1,10 @@
 INCLUDE_QT = -I$(QTDIR)/include
-LIB_QT = -L$(QTDIR)/lib
+LIB_QT = -L$(QTDIR)/lib -Wl,-R$(QTDIR)/lib
 
 all: icefonsel icecolsel
 
 icefonsel: icefonsel.cpp
-	g++ $(INCLUDE_QT) $(LIB_QT) -lqt icefonsel.cpp -o icefonsel
+	g++ $(INCLUDE_QT) $(LIB_QT) -lqt-mt icefonsel.cpp -o icefonsel
 
 icecolsel: icecolsel.cpp
-	g++ $(INCLUDE_QT) $(LIB_QT) -lqt icecolsel.cpp -o icecolsel
+	g++ $(INCLUDE_QT) $(LIB_QT) -lqt-mt icecolsel.cpp -o icecolsel
